apps/src/templates/teacherDashboard/SectionActionDropdown.jsx

```diff
@@ -115,24 +115,19 @@ class SectionActionDropdown extends Component {
       <span>
         <QuickActionsCell>
           <PopUpMenu.Item
-            href={pegasus(`/teacher-dashboard#/sections/${sectionData.id}`)}
+            href={teacherDashboardUrl(sectionData.id, '/progress')}
```

Comment:
I vaguely remember encountering problems with unit tests and stories when referencing experiments inside react component render methods, but if tests are passing then this seems fine.